packages/astro/src/@types/astro.ts

```diff
@@ -1686,6 +1686,47 @@ export interface AstroUserConfig {
 	 */
 	legacy?: object;
 
+	/**
+	 * @docs
```

Comment:
I think this should be moved much higher up in the file? It loos like this will now appear under "Legacy Flags" in the sidebar.

This is a top-level option, right? So it should probably move up maybe between `scopedStyleStrategy` and `vite`? These aren't in alphabetical order, but manually by common usage first, and then whenever just seems right. I think just above `vite` makes sense, but open to suggestions if you don't like that!

Reply:
Done!